spec/features/work_packages/is_empty_filter_spec.rb

```diff
@@ -0,0 +1,111 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2023 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+require 'spec_helper'
+
+RSpec.describe 'Filter by date with "is empty"', js: true do
```

Comment:
There's a new Rubocop well... - cop - that suggests the symbol style metadata. Also, using cuprite add a small performance boost to this spec out of the box :)

```suggestion
RSpec.describe 'Filter by date with "is empty"', :js, :with_cuprite do
```

Reply:
Hey @aaron-contreras, what does this mean?

> There's a new Rubocop well... - cop - that suggests the symbol style metadata.

Reply:
`js: true` can also be written as `:js`, as RSpec picks up the example group and infers you'd like to set it to true. The style that RSpec is enforcing now is the latter. It also reads a little more declaratively that way :)

Reply:
Aah okay, thanks!